src/screens/ProfileStack.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { createStackNavigator } from '@react-navigation/stack'
+import { useSelector } from 'react-redux'
+import styles from '../styles/styles'
+import CommentSection from '../components/CommentSection'
+import HomeScreen from '../components/HomeScreen'
+import SinglePost from '../components/SinglePost'
+import { Ionicons } from '@expo/vector-icons'
```

Comment:
You have a lot of unnecessary imports here

Reply:
This was fixed in the last commit! :)